msgs/proto/posts.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+// openboard
+//
+// user semantics:
+//
+//  Add{T}(s)    (Add{T}(s)Req)    returns {T}(s)Resp // POST
+//  Ovr{T}(s)    (Ovr{T}(s)Req)    returns {T}(s)Resp // PUT
+//  Mod{T}(s)    (Mod{T}(s)Req)    returns {T}(s)Resp // PATCH
+//  Get{T}       (Get{T}Req)       returns {T}Resp    // GET
+//  Fnd{T}s      (Fnd{T}sReq)      returns {T}sResp   // GET
+//  Rmv{T}(s)    (Rmv{T}(s)Req)    returns EmptyResp  // DELETE
+//  Unr{T}       (Unr{T}Req)       returns {T}Resp    // PATCH
+package pb;
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+message metadata {
+	string type = 1;
+	// ?? other things ??
+}
+
+message AddPostReq {
+	string title = 1;
+	string body = 2;
+	metadata metadata = 3;
+}
+
+message PostResp {
+	uint32 id = 1;
+	string link = 2;
+	google.protobuf.Timestamp created = 3;
+    google.protobuf.Timestamp updated = 4;
+    google.protobuf.Timestamp deleted = 5;
+    google.protobuf.Timestamp blocked = 6;
```

Comment:
What would be the use of this blocked field, and how would a post be considered blocked?

Reply:
I suppose an admin could block a post by setting this date. Much like you would archive a post by setting the `deleted` field.

Reply:
... or this is just a copy/paste remnant from [user.proto](https://github.com/OpenEugene/openboard/blob/master/msgs/proto/user.proto)

Reply:
copy pasta! yum!